### PR TITLE
chore(github-pr-template): spellcheck and link fix

### PR DESCRIPTION
**Pull request by C0ZEN**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes

## Does this PR introduce a breaking change?

- [x] No

## Describe the changes

Github issue number or Jira issue URL: N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
 
 ## Does this PR introduce a breaking change?
 
-read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)
+<!-- Read [what is a breaking change?](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-) -->
```
Comment (tiloyi) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: This text should not be commented on.
Can you remove the `<!-- -->` please?

Comment (C0ZEN) [reply .github/PULL_REQUEST_TEMPLATE.md]: @tiloyi what about having it directly into the title?

Comment (tiloyi) [reply .github/PULL_REQUEST_TEMPLATE.md]: Indeed, it seems good to me to do that :)

**Comment by C0ZEN** [comment]
@tiloyi ready

**Pull request merged**